pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -429,31 +429,21 @@ func assertSSTContents(sst []byte, sstTimestamp hlc.Timestamp, stats *enginepb.M
 
 	// Check SST KV pairs.
 	iter.SeekGE(storage.MVCCKey{Key: keys.MinKey})
```

Comment:
Looks like leftover since we have it as part of for loop now.

Reply:
Oops, nice catch -- thank you!